README.md

```diff
@@ -1,20 +1,91 @@
 # Ruby Class Variables And Class Methods Lab
 
-## Outline
+## Objectives
 
+1. Use class variables to keep track of data pertaining to a class. 
+2. Define class methods to expose data pertaining to a class.
+
+## Overview
+
+In this lab, we'll be dealing with a `Song` class. The `Song` class can produce individual songs. Each song has a name, an artist and a genre. We need our `Song` class to be able to keep track of the number of songs that it creates. 
+
+```ruby
 Song.count
+  => 30
```

Comment:
Prefix the `=>` with a `#` so if they copy paste this code block into `irb` it will work